Updgrade the Bytewax code to 0.19.1

awmatheson opened the pull request:
Upgrades to the most recent API.

awmatheson committed:
Update to 0.19

Updates the bytewax example to 0.19.1

awmatheson committed:
Update requirements.txt

awmatheson committed:
Update requirements.txt

Mikkolehtimaki committed:
add missing requirements

Mikkolehtimaki reviewed (commented):
Thanks Zander, a much appreciated update! I added a couple of requirement versions. The expected behavior would be to keep bytewax running and processing the log once the `bytewax/iot.py` script is run. I think that's not happening after the update, could you confirm?

awmatheson committed:
tail

awmatheson commented:
> Thanks Zander, a much appreciated update! I added a couple of requirement versions. The expected behavior would be to keep bytewax running and processing the log once the `bytewax/iot.py` script is run. I think that's not happening after the update, could you confirm?

Good catch, thanks! Changed it to tail
